support/packages/github-to-jira-synchronization-tool/src/config/config.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * SPDX-FileCopyrightText: © 2020 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: MIT
+ */
+
+const fs = require('fs');
+const path = require('path');
+
+let config = null;
+
+function loadConfig() {
+	if (config) {
+		return config;
+	}
+
+	try {
+		const json = fs.readFileSync(
```

Comment:
:thinking: I think you can use `require()` for this

Reply:
It would save you the readFileSync and the JSON.parse

Reply:
I though that using `require` with json files may have some problems. But maybe it is just in case the json is dynamic? and how require caches stuff? 🤔

Reply:
Yeah, require() caches forever unless you explicitly clean it by hand. In any case, this is static as it is inside the source code so it's all good.

I would say a good criterion is:

1. Use require if the JSON is a static project resource (like in this case)
2. Use readFile in case the JSON is outside of the project and it may change or if there are myriads of JSON files to read and you don't want to cache them
3. Use require if the JSON is outside of the project but it won't change during the life of the process and you want to cache it (for example: I usually read the package.json file from liferay-npm-bundler because there's only one and it doesn't change while liferay-npm-bundler is executed. Also, because I access it from many places in the code, I take advantage of require()'s cache.